### DISCUSSION TITLE
Admonitions in `dd` don't look right on Chrome (Android) w/ larger default font-sizes

### DISCUSSION THREAD

**Discussion by pradyunsg** [pradyunsg/furo#416] [Potential Issue]
**Describe the bug**
The font sizing seems to be off.

**To Reproduce**

1. Go to https://pradyunsg.me/furo/reference/admonitions/ on Chrome for Android, with non-default base-font size.
2. Scroll down to the list of admonition's supported types.
3. See weird title sizing.

**Expected behavior**

The admonitions in the admonition demo have the correct font-size, that's not relative to the admonition's font size, instead of the page's font-size.

**Device**

- Browser: Chrome on Android
- Device: @webknjaz's phone. :P

  <details><summary>It's a custom MIUI packaged by xiaomi.eu, looks like there's Android 10 underneath</summary>

  ![IMAGE 2020-10-05 18:06:24](https://user-images.githubusercontent.com/3275593/95080274-7b67e680-0735-11eb-9f29-466f5f53614d.jpg)

  ![IMAGE 2020-10-05 18:12:09](https://user-images.githubusercontent.com/3275593/95080805-48722280-0736-11eb-9867-fe8a32af4a81.jpg)

  </details>

**Additional context**

This seems to be due to Chrome on Android taking the 100% font-size to be relative to the "base" font size (i.e. the one set in `html`) instead of the one set in the parent element.

<details>
<summary><b>Screenshots</b></summary>

![IMAGE 2020-10-05 18:03:34](https://user-images.githubusercontent.com/3275593/95080019-1613f580-0735-11eb-8f22-a602f9c16826.jpg)

![IMAGE 2020-10-05 18:03:10](https://user-images.githubusercontent.com/3275593/95079989-07c5d980-0735-11eb-886e-a2cbf9f48399.jpg)

</details>

**Comment by pradyunsg** [comment]
Fixed in 393c59b32793c5f62c52e5a215edad79b494795b, as far as I can tell. I'll let @webknjaz confirm. :)

**Comment by webknjaz** [comment]
I wish :(

**Comment by webknjaz** [comment]
Looks like it's necessary to do something like
```diff
-    --admonition-title-font-size: 0.8125rem;
+    --admonition-title-font-size: 0.8125em;
```

**Comment by pradyunsg** [comment]
That leaves the admonition body at the wrong size though, right?

**Comment by webknjaz** [comment]
I think so
